src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './checkbox.css';
+import { useFormContext } from 'react-hook-form';
+
+interface Props {
+  name: string;
+  description: string;
+}
+
+function Checkbox({ name, description }: Props) {
+  const { register, formState } = useFormContext();
+  const { errors } = formState;
+  const checkboxRef: React.RefObject<HTMLInputElement> = React.createRef();
+  const checked = checkboxRef.current ? checkboxRef.current.checked : false;
```

Comment:
you should use [Optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#syntax)

```suggestion
  const checked = checkboxRef.current?.checked;
```